### PR TITLE
docs: List qStudio in Integrations

### PR DESCRIPTION
**Pull request by richb-hanover**
qStudio is cool. I think I have mentioned it all the places it deserves to be

### CODE REVIEW

File: web/book/src/project/integrations/qstudio.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# qStudio IDE
+
+qStudio is a SQL GUI that lets you browse tables, run SQL scripts, and chart and
```
Comment (ryanhamilton) [inline web/book/src/project/integrations/qstudio.md:3]: 👍 Perfect.

**Comment by max-sixty** [comment]
Nice!

Are there docs on how to get prqlc working in qstudio? IIUC someone needs a `prqlc` binary in their path?

**Comment by richb-hanover** [comment]
Good catch. I added a note to the qstudio.md page reminding people of this

File: web/book/src/SUMMARY.md [hunk 2]
```diff
@@ -103,15 +103,16 @@ General information about the project, tooling and development.
...
+  - [qStudio](./project/integrations/qstudio.md)
+  - [ClickHouse](./project/integrations/clickhouse.md)
   - [DuckDB](./project/integrations/duckdb.md)
   - [Prefect](./project/integrations/prefect.md)
   - [VS Code](./project/integrations/vscode.md)
   - [Rill](./project/integrations/rill.md)
-  - [Syntax highlighting](./project/integrations/syntax-highlighting.md)
   - [PostgreSQL](./project/integrations/postgresql.md)
+  - [`prqlc CLI`](./project/integrations/prqlc-cli.md)
+  - [Syntax highlighting](./project/integrations/syntax-highlighting.md)
```
Comment (max-sixty) [inline web/book/src/SUMMARY.md:116]: Can we restore the old order and put qStudio under DuckDB? We should have the native integrations higher than those which require add-ons... (same on website)

Comment (richb-hanover) [reply web/book/src/SUMMARY.md:116]: This is the order of integrations in the SUMMARY.md. Would you sort them into the order that makes most sense? (I don't have a clue about which to put where, or which are most important to us (and therefore deserve to be at the top)) Thanks

- [Integrations](./project/integrations/README.md)

  - [Jupyter](./project/integrations/jupyter.md)
  - [qStudio](./project/integrations/qstudio.md)
  - [ClickHouse](./project/integrations/clickhouse.md)
  - [DuckDB](./project/integrations/duckdb.md)
  - [Prefect](./project/integrations/prefect.md)
  - [VS Code](./project/integrations/vscode.md)
  - [Rill](./project/integrations/rill.md)
  - [PostgreSQL](./project/integrations/postgresql.md)
  - [`prqlc CLI`](./project/integrations/prqlc-cli.md)
  - [Syntax highlighting](./project/integrations/syntax-highlighting.md)

Comment (max-sixty) [reply web/book/src/SUMMARY.md:116]: The existing one was good I thought! This adds in qStudio

```diff
[12. Integrations](https://prql-lang.org/book/project/integrations/index.html)

    [12.1. prqlc CLI](https://prql-lang.org/book/project/integrations/prqlc-cli.html)
    [12.2. ClickHouse](https://prql-lang.org/book/project/integrations/clickhouse.html)
    [12.3. Jupyter](https://prql-lang.org/book/project/integrations/jupyter.html)
    [12.4. DuckDB](https://prql-lang.org/book/project/integrations/duckdb.html)
+ qStudio
    [12.5. Prefect](https://prql-lang.org/book/project/integrations/prefect.html)
    [12.6. VS Code](https://prql-lang.org/book/project/integrations/vscode.html)
    [12.7. Rill](https://prql-lang.org/book/project/integrations/rill.html)
-   [12.8. Syntax highlighting](https://prql-lang.org/book/project/integrations/syntax-highlighting.html)
    [12.9. PostgreSQL](https://prql-lang.org/book/project/integrations/postgresql.html)
+   [12.8. Syntax highlighting](https://prql-lang.org/book/project/integrations/syntax-highlighting.html)
```

(prob syntax highlighting should be at the bottom, not required for this PR...)

Comment (richb-hanover) [reply web/book/src/SUMMARY.md:116]: Done. It occurred to me to separate the "external "third-party" tools from the "internal-ish" ones like `prqlc` and the syntax highlighting. They felt like different audiences. But I'm totally OK with this presentation

File: web/website/content/_index.md [hunk 3]
```diff
@@ -175,6 +175,14 @@ integrations_section:
         a database. It can also set up an in-memory DuckDB instance, populated
         with a pandas DataFrame."
 
+    - label: "qStudio"
```
Comment (max-sixty) [inline web/website/content/_index.md:189]: Same ordering here if possible!

File: web/book/src/project/integrations/README.md [hunk 4]
```diff
@@ -2,12 +2,13 @@
...
+- [Jupyter](./jupyter.md)
 - [DuckDB](./duckdb.md)
+- [qStudio](./qstudio.md)
 - [Prefect](./prefect.md)
 - [VS Code](./vscode.md)
 - [Rill](./rill.md)
 - [PostgreSQL](./postgresql.md)
-
-We also have a CLI, [`prqlc`](./prqlc-cli.md)
+- [Syntax highlighting](./syntax-highlighting.md) for many popular tools.
```
Comment (max-sixty) [inline web/book/src/project/integrations/README.md:15]: Thanks for syncing these!

(kinda agree re the CLI, though only one so it doesn't really make a category...)

**Comment by max-sixty** [comment]
Thanks @richb-hanover !

**Pull request merged**